doc/source/metadata/Introduction.rst

```diff
@@ -0,0 +1,724 @@
+    holding WMO identifier and Met Office identifiers, respectively.
+
+.. how tied to the code are these identifiers, as they may not be appropiate
+    for other users?
+
+
+Variables
+^^^^^^^^^
+
+The majority of variables are the same for both gridded data and spot data.
+Only the differences will be discussed here.
+The variable attributes are common to the gridded data, so these will not be discussed. 
+
+.. literalinclude:: temp12max_spotperc_ncdump.txt
+    :tab-width: 4
+    :lines: 18-23, 43-50, 57-58
+
+spot_index
+    An arbitrary integer index for the sites. 
+
+altitude
+    Height of site above sea level. 
+
+latitude, longitude
+    Site positions are only provided in latitude and longitude,
+    regardless of the projection used for the corresponding grid.
+    These can be considered as relative the WGS84 or the World Geodetic System 1984 datum,
+    although this is not explicit in the metadata. 
+
+wmo_id
+    For WMO sites, this is the numerical WMO identifier stored as a string; 
+    for other sites this is a set to ``NaN`` (at present, the formatting of this is not ideal, but will be changed to ensure that the numbers are right-justified and padded with zeros in the next release). 
+
+Special parameters
+------------------
+
+Weather code
+************
+
+At present, a single weather code is offered,
+which is derived from the full set of probabilistic data. 
+For this reason, some of the metadata are slightly different. 
+These difference are described below, but the full ncdump listing
+is shown in an appendix.
+
+As this is deterministic, there is no dimension and coordinate variable for 
+``threshold`` or ``percentile``. 
+
+There are two new coordinate variables.
+
+.. literalinclude:: wx_code_ncdump.txt
+    :tab-width: 4
+    :lines: 11-24
+
+weather_code
+    Is the integer code used to represent the weather type in the main variable data. 
+
+weather_code_meaning
+    Provides corresponding short descriptions of each weather type in the weather_code list. 
+
+Wind direction
+**************
+
+At present, probabilities and percentiles are not offered for wind direction, 
+just a single set of deterministic values, which are a mean across realizations
+(ensemble members) from a single model. 
+For this reason, some of the metadata are slightly different 
+from the nearest equivalent percentiles. 
+These difference are described below, but the full ncdump listing
+is shown in an appendix.
+
+.. note::
+
+    In future, we expect to at least be able to generate 
+    percentiles of wind direction.
+
+
+The global attribute ``mosg__model_configuration`` is always a single model 
+(for the Met Office ``uk_ens`` or ``gl_ens`` 
+
+For gridded data only, the ``title`` only identifies the single model used
+(for the Met Office, this would be either
+``MOGREPS-UK Model Forecast on UK 2 km Standard Grid`` or 
+``MOGREPS-G Model Forecast on UK 2 km Standard Grid``),
```

Comment:
These are out of date. The titles now include the `Post-processed` prefix to avoid issues with visual weather distinguishing between the raw model output and the IMPROVER products. We have done processing here, so the title is not unreasonably applied.

Reply:
Thanks. It's in my sample data, but forgot about the change.